1.3/spec.md

```diff
@@ -0,0 +1,381 @@
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in
+this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+## Abstract
+
+MBTiles is a specification for storing tiled map data in
+[SQLite](http://sqlite.org/) databases for immediate usage and for transfer.
+MBTiles files, known as **tilesets**, MUST implement the specification below
+to ensure compatibility with devices.
+
+## Compatibility
+
+*This section is informative and does not add requirements to implementations.*
+
+Because views may be used to produce the MBTiles schema, two implementations
+may store tiles with different internal details, meaning one implementation
+may not be able to add to an existing file.
+
+As a container format, MBTiles can store any tiled data, so data can be stored
+that an implementation cannot do anything with.
+
+Relying on metadata keys not defined in the specification can cause
+compatibility problems.
+
+## Database Specifications
+
+Tilesets SHALL be valid SQLite databases of
+[version 3.0.0](http://sqlite.org/formatchng.html) or higher.
+Only core SQLite features are permitted; tilesets SHALL NOT require extensions.
+
+MBTiles databases MAY use [the officially assigned magic number](http://www.sqlite.org/src/artifact?ci=trunk&filename=magic.txt)
+to be easily identified as MBTiles.
+
+## Database
+
+Note: the schemas outlined are meant to be followed as interfaces.
+SQLite views that produce compatible results MAY be used instead.
+For convenience, this specification refers to tables and virtual
+tables (views) as tables.
+
+## Charset
+
+All text in `text` columns of tables in an mbtiles tileset MUST be encoded as UTF-8.
+
+### Metadata
+
+#### Schema
+
+The database MUST contain a table or view named `metadata`.
+
+This table or view MUST yield exactly two columns of type `text`, named `name` and
+`value`. A typical create statement for the `metadata` table:
+
+    CREATE TABLE metadata (name text, value text);
+
+#### Content
+
+The metadata table is used as a key/value store for settings. It MUST contain these two rows:
+
+* `name` (string): The human-readable name of the tileset.
+* `format` (string): The file format of the tile data: `pbf`, `jpg`, `png`, `webp`, or an [IETF media type](https://www.iana.org/assignments/media-types/media-types.xhtml) for other formats.
+
+`pbf` as a `format` refers to gzip-compressed vector tile data in
+[Mapbox Vector Tile](https://github.com/mapbox/vector-tile-spec/) format.
+
+The `metadata` table SHOULD contain these four rows:
+
+* `bounds` (string of comma-separated numbers): The maximum extent of the rendered map area. Bounds must define an
+  area covered by all zoom levels. The bounds are represented as `WGS 84`
+  latitude and longitude values, in the OpenLayers Bounds format
+  (left, bottom, right, top). For example, the `bounds` of the full Earth, minus the poles, would be:
+  `-180.0,-85,180,85`.
```

Comment:
I assume the `-180.0,-85,180,85` is supposed to be web mercator format. However, should the bounds actually be `-180.0,-85.06,180.0,85.06` which is `ESPG:3857`?